// @google-cloud/bigquery/build/src/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/*!
 * Copyright 2019 Google Inc. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
var bigquery_1 = require("./bigquery");
exports.BigQuery = bigquery_1.BigQuery;
exports.BigQueryDate = bigquery_1.BigQueryDate;
exports.BigQueryDatetime = bigquery_1.BigQueryDatetime;
exports.BigQueryTime = bigquery_1.BigQueryTime;
exports.BigQueryTimestamp = bigquery_1.BigQueryTimestamp;
exports.Geography = bigquery_1.Geography;
var dataset_1 = require("./dataset");
exports.Dataset = dataset_1.Dataset;
var job_1 = require("./job");
exports.Job = job_1.Job;
var table_1 = require("./table");
exports.Table = table_1.Table;
var routine_1 = require("./routine");
exports.Routine = routine_1.Routine;
var model_1 = require("./model");
exports.Model = model_1.Model;
